core/AutoValue_Test.java

```diff
@@ -0,0 +1,34 @@
+
+
```

Comment:
nit:empty lines. Should this have a package or is this following what google does?

Reply:
Hmm I didn't touch this file, I didn't notice it in the diff before. I'll figure out what happened here